src/components/probabilityCards.js

```diff
@@ -1,13 +1,7 @@
-AFRAME.registerComponent("probability", {
+AFRAME.registerComponent("probability-cards", {
```

Comment:
I think this file is going to have other logic involved in it, so it'd be better to leave it as `probability`